astropy/visualization/wcsaxes/tests/test_misc.py

```diff
@@ -499,6 +500,41 @@ def test_simplify_labels_usetex(ignore_matplotlibrc, tmp_path):
     fig.savefig(tmp_path / "plot.png")
 
 
+@pytest.mark.parametrize(
+    "usetex, unicode_minus, label_str",
+    [
+        (True, True, "$-{}$"),
+        (True, False, "$-{}$"),
+        (False, True, "\N{MINUS SIGN}{}"),
+        (False, False, "-{}"),
+    ],
+)
+def test_simplify_labels_minus_sign(
+    ignore_matplotlibrc, usetex, unicode_minus, label_str
+):
+    # Ensure minus signs aren't removed from the front of labels across a grid of configuration possibilities
+    if usetex and TEX_UNAVAILABLE:
+        pytest.skip("TeX is unavailable")
+    ticklabels = TickLabels(None)
+    with mpl.rc_context(
+        rc={"text.usetex": usetex, "axes.unicode_minus": unicode_minus}
+    ):
+        expected_labels = []
+        for i in [1, 2]:
+            label = label_str.format(i)
+            ticklabels.add(
+                axis="axis",
+                world=0,
+                angle=0,
+                text=label,
+                axis_displacement=0,
+                data=(i, i),
+            )
+            expected_labels.append(label)
+        ticklabels.simplify_labels()
```

Comment:
actually I expect this line would be the only one that needs to happen within the rc_context, or am I missing something ?

Reply:
No, you're right!